openmc/settings.py

```diff
@@ -125,6 +126,8 @@ class SettingsFile(object):
         Coordinates of the lower-left point of the UFS mesh
     ufs_upper_right : tuple or list
         Coordinates of the upper-right point of the UFS mesh
+    resonance_scattering : ResonanceScattering or iterable thereof
+        The elastic scattering model to use for resonant isotopes.
```

Comment:
Remove the period on this docstring, also for consistency (I really don't care one way or the other, as long as we are consistent)